packages/ui-components/src/components/SelectRow/SelectRow.stories.js

```diff
@@ -6,9 +6,8 @@
 import React, { useState, useEffect } from "react"
 import PropTypes from "prop-types"
 
-import { SelectRow } from "./index.js"
-import { SelectOption } from "../SelectOption/SelectOption.component"
-import { Default as DefaultSelectOptionStory } from "../SelectOption/SelectOption.stories"
+import SelectRow from "./SelectRow.component"
+import { SelectOption } from "../../deprecated_js/SelectOption/SelectOption.component"
 import { PortalProvider } from "../../deprecated_js/PortalProvider/PortalProvider.component"
 
 export default {
```

Comment:
Do we've a future task to remove this component once users switch over? Let's discuss this offline, how you handle such deprecations in the team. 😊

Reply:
I mean yes for sure we have to remove them